File: base/src/main/java/me/shetj/base/base/BaseMessage.java
package me.shetj.base.base;

import android.support.annotation.Keep;

/**
 * <b>@packageName：</b> me.shetj.base.base<br>
 * <b>@author：</b> shetj<br>
 * <b>@createTime：</b> 2018/3/28<br>
 * <b>@company：</b><br>
 * <b>@email：</b> 375105540@qq.com<br>
 * <b>@describe</b><br>
 */
@Keep
public class BaseMessage <T>{
	public int type;
	public int position;
	public T obj;
	public String msg;
}

File: base/src/main/java/me/shetj/base/base/BaseMessage.kt
package me.shetj.base.base

import androidx.annotation.Keep

/**
 * **@packageName：** me.shetj.base.base<br></br>
 * **@author：** shetj<br></br>
 * **@createTime：** 2018/3/28<br></br>
 * **@company：**<br></br>
 * **@email：** 375105540@qq.com<br></br>
 * **@describe**<br></br>
 */
@Keep
class BaseMessage<T> {
    var type: Int = 0
    var position: Int = 0
    var obj: T? = null
    var msg: String? = null
}
